packages/widgets/src/menu.ts

```diff
@@ -1175,7 +1244,7 @@ export namespace Menu {
         {
           className,
           dataset,
-          tabindex: '0',
+          tabindex: '-1',
```

Comment:
In other words, the user will navigate to menu items via the left/right arrow keys?